ui/src/pages/help-center/HelpCenter.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button, CardContent, Typography, Box, Card } from '@mui/material';
+import {
+  Support,
+  ForumOutlined,
+  DatasetOutlined,
+  NorthEast,
+  SaveAlt,
+  MapOutlined,
+} from '@mui/icons-material';
+import { KnowledgeBaseIcon } from 'icons';
+import { Page } from 'components/page';
+import { FC, ReactNode, useCallback } from 'react';
+import { Messages } from './HelpCenter.messages';
+import { CardsData, cardIds, startIcon } from './HelpCenter.constants';
+import { useUser } from 'contexts/user';
+
+export const HelpCenter: FC = () => {
+  const { user } = useUser();
+
+  const getIcon = useCallback((cardId: string): ReactNode => {
+    switch (cardId) {
+      case cardIds.pmmDocs:
+        return <KnowledgeBaseIcon />;
+      case cardIds.support:
+        return <Support />;
+      case cardIds.forum:
+        return <ForumOutlined />;
+      case cardIds.pmmDump:
+        return <DatasetOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const getButtonStartIcon = useCallback((iconName: string): ReactNode => {
+    switch (iconName) {
+      case startIcon.download:
+        return <SaveAlt />;
+      case startIcon.map:
+        return <MapOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const shouldDisplayCard = useCallback(
+    (itemId: string): boolean =>
+      !(
+        !user?.isPMMAdmin &&
+        (itemId === cardIds.pmmDump || itemId === cardIds.pmmLogs)
+      ),
+    [user]
+  );
+
+  return (
+    <Page title={Messages.pageTitle}>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: {
+            xs: '1fr',
+            sm: '1fr',
+            md: 'repeat(2, 1fr)',
+          },
+          gap: 4,
+        }}
+      >
+        {CardsData.map(
+          (item) =>
+            shouldDisplayCard(item.id) && (
```

Comment:
I think it may be good to extract the card into it's own component. Then you could do a check like this:

```tsx
const Card: FC = (props) => {
 if (!shouldDisplayCard(id)) {
    return null;
 }

  return <Card .... />
}
```